android-static-binding-generator/run-tests.js

```diff
@@ -0,0 +1,36 @@
+(function () {
+    let { spawnSync, execSync } = require('child_process');
+
+    console.log("Installing JavaScript parser test dependencies.");
+
+    let npmInstallResult = execSync('npm install');
+
+    console.log(npmInstallResult.toString());
+
+    console.log("Executing JavaScript parser tests using Jasmine.");
+
+    const subprocess = spawnSync('node', ['./node_modules/jasmine/bin/jasmine.js', 'JASMINE_CONFIG_PATH=jasmine.json'], { cwd: __dirname });
+
+    let stdout = subprocess.stdout.toString();
+    let stderr = subprocess.stderr.toString();
+    let code = subprocess.status;
+    let err = subprocess.error;
+
+    if (code !== 0) {
+        console.log("Tests runner exited with a non-zero code.");
+
+        let errorString = stdout;
+
+        if (err) {
+            errorString = err;
+        }
+
+        if (stderr) {
+            errorString = stderr;
```

Comment:
don't override error string, but rather throw an object with both the `supbrpocess.error` and  `subprocess.stderr` so we don't loose information

Reply:
Is it possible that both are set at the same time though? My understanding is that `subprocess.error` is raised when something goes wrong while spawning the process, whereas `stderr` is a result of an error thrown inside the process.

Reply:
To be honest, I'm not sure if both can be present. It's just a suggestion when I look at the code.

Reply:
Throwing an object results in `[object Object]` in the logs, which is not very helpful.